docset/winserver2022-ps/failoverclusters/Set-ClusterOwnerNode.md

```diff
@@ -32,21 +32,21 @@ failure of a resource or a clustered role.
 
 ### Example 1
 
-```
-PS C:\> Get-ClusterResource -Name "Cluster Disk 3" | Set-ClusterOwnerNode -Owners node1,node2
+```powershell
+Get-ClusterResource -Name "Cluster Disk 3" | Set-ClusterOwnerNode -Owners node1,node2
 ```
 
-This example sets the possible owners for cluster named Cluster Disk 3 on the local cluster to the
-nodes named node1 and node2.
+This example sets the possible owners for cluster named `Cluster Disk 3` on the local cluster to the
```

Comment:
Here the docs say `possible` owners, and down on line 48 it says `preferred`.

Reply:
This is correct. If the input object is a cluster resource, then the parameter applies the possible owner. Preferred owners only apply to cluster groups.